components/motor/gpio/basic_test.go

```diff
@@ -95,6 +95,10 @@ func TestMotorABPWM(t *testing.T) {
 		test.That(t, mustGetGPIOPinByName(b, "2").Get(context.Background()), test.ShouldEqual, true)
 		test.That(t, mustGetGPIOPinByName(b, "3").PWM(context.Background()), test.ShouldEqual, .5)
 
+		// It's only okay to have 0 RPM if you want to move 0 revolutions.
+		test.That(t, m.GoFor(ctx, 0, 0, nil), test.ShouldBeNil)
+		test.That(t, m.GoFor(ctx, 0, 1, nil), test.ShouldBeError, errors.New("rpm must be nonzero for motor to move"))
```

Comment:
If you're expecting to use this error for other parts of the motor package, it may make sense to make it a general error such as
` var  errMoveWithZeroRPM = error.New("your error")`